libcodechecker/analyze/analyzer_crash_handler.py

```diff
@@ -12,13 +12,14 @@
 
 from libcodechecker.logger import LoggerFactory
 
+LOG = LoggerFactory.get_new_logger('ANALYZER_CRASH_HANDLER')
```

Comment:
Remove the `_`s from the logger name. Most logger tags have been fixed earlier to not include underscores.